packages/test-utils/src/mocks/localstorage.ts

```diff
@@ -1,9 +1,23 @@
-export function mockLocalStorage(value: string) {
-  Object.defineProperty(window, 'localStorage', {
+export function mockLocalStorage(initialStore: { [key: string]: string }) {
+  let store = initialStore;
```

Comment:
this is ok but could lead to miss interpretation, remember that jest don't share process between tests.